Model/Adminhtml/Page/Source/WordpressPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mooore\WordpressIntegrationCms\Model\Adminhtml\Page\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Message\ManagerInterface;
+use Mooore\WordpressIntegrationCms\Model\RemotePageRepository;
+
+class WordpressPage implements OptionSourceInterface
+{
+    /**
+     * @var RemotePageRepository
+     */
+    private $remotePageRepository;
+    /**
+     * @var ManagerInterface
+     */
+    private $messageManager;
+
+    public function __construct(RemotePageRepository $remotePageRepository, ManagerInterface $messageManager)
+    {
+        $this->remotePageRepository = $remotePageRepository;
+        $this->messageManager = $messageManager;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function toOptionArray()
+    {
+        $options = [];
```

Comment:
Even nitpicken hoor; Is deze declaratie nodig?

Reply:
Done :+1: